packages/bridge-ui/src/utils/recommendProcessingFee.ts

```diff
@@ -19,12 +21,15 @@ export async function recommendProcessingFee(
   signer: Signer,
 ): Promise<string> {
   if (!toChain || !fromChain || !token || !signer || !feeType) return '0';
-  const provider = providers[toChain.id];
-  const gasPrice = await provider.getGasPrice();
+
+  const destProvider = providers[toChain.id];
+  const gasPrice = await destProvider.getGasPrice();
```

Comment:
we use legacy type 0 transactions here, but since both chains now support EIP1559, I think we should move towards implementing the more modular/predictable EIP1559 transactions in the future. Doesnt need to be done in this PR though.

Reply:
Noted. Thanks